src/relay/transforms/simplify_expr.cc

```diff
@@ -160,7 +160,7 @@ class SimplifyConsecutiveCast : public DFPatternRewrite {
   DFPattern cast1_;
 };
 
-bool CheckDataTypeMaxMinValue(DataType dtype, double min_value, double max_value) {
+bool CheckDataTypeMaxMinValue(DataType dtype, double min_value, double max_value, int mode = 0) {
```

Comment:
Please comment the meaning of different `mode`

Reply:
Done